bitcoin/src/address/mod.rs

```diff
@@ -115,14 +135,27 @@ pub trait NetworkValidation: sealed::NetworkValidation + Sync + Send + Sized + U
     const IS_CHECKED: bool;
 }
 
+/// Marker trait for `FromStr` and `serde::Deserialize`.
+///
+/// This allows users to use `V: NetworkValidation` in conjunction with derives. Is only ever
+/// implemented for `NetworkUnchecked`.
+pub trait NetworkValidationUnchecked:
+    NetworkValidation + sealed::NetworkValidationUnchecked + Sync + Send + Sized + Unpin
```

Comment:
Unless I'm missing something, those other bounds should be just impls on `Address`. (But `Sized` is weird.)

Anyway, you forgot `UnwindSafe` and `RefUnwindSafe`.

Reply:
Yeah. Other than the sealed one and `NetworkValidation` it's not clear that these are providing value?

Reply:
#3974